buildSrc/build.gradle

```diff
@@ -77,7 +77,10 @@ tasks.withType(JavaCompile).configureEach {
  *****************************************************************************/
 
 repositories {
-  jcenter()
+  mavenCentral()
+  maven {
```

Comment:
Can't we just use `gradlePluginPortal()` here?

Reply:
fixed